utils/build-presets.ini

```
@@ -1301,6 +1301,8 @@ extra-cmake-options=
    -DCMAKE_C_FLAGS="-gline-tables-only"
    -DCMAKE_CXX_FLAGS="-gline-tables-only"
 
+dsymutil-args="--verify-dwarf=none"
```

The flag has been renamed in the original PR

```suggestion
extra-dsymutil-args="--verify-dwarf=none"
```